src/main/kotlin/org/rust/lang/core/macros/MacroExpander.kt

```diff
@@ -184,15 +184,20 @@ class MacroExpander(val project: Project) {
                     }
                 }
                 else -> {
-                    sb.append(" ")
-                    sb.append(child.text)
-                    sb.append(" ")
+                    sb.safeAppend(child.text)
                 }
             }
         }
         return true
     }
 
+    private fun StringBuilder.safeAppend(str: String) {
```

Comment:
`safeAppend` name is not worse than `appendWrapped` or `appendWrappedWithWhitespace` because all these variants don't say that this method does.
Due to I don't believe that there is a better name I just suggest to add some short explanation in comment or javadoc